staging/src/slime.io/slime/modules/foo/Dockerfile

```diff
@@ -0,0 +1,7 @@
+# Use distroless as minimal base image to package the manager binary
```

Comment:
this module (name) will confuse the users

Reply:
removed